src/americana.js

```diff
@@ -23,27 +22,11 @@ import { default as OpenMapTilesSamples } from "openmapsamples/samples/OpenMapTi
 export function buildStyle() {
   var getUrl = window.location;
   var baseUrl = getUrl.protocol + "//" + getUrl.host + getUrl.pathname;
-
-  return {
-    id: "streets",
-    name: "Americana",
-    glyphs:
-      "https://openhistoricalmap.github.io/map-styles/fonts/{fontstack}/{range}.pbf",
-    layers: Layers.build(Label.getLocales()),
-    sources: {
-      openmaptiles: {
-        url: config.OPENMAPTILES_URL,
-        type: "vector",
-      },
-    },
-    sprite: new URL("sprites/sprite", baseUrl).href,
-    light: {
-      anchor: "viewport",
-      color: "white",
-      intensity: 0.12,
-    },
-    version: 8,
-  };
+  return Style.build(
+    config.OPENMAPTILES_URL,
+    new URL("sprites/sprite", baseUrl).href,
```

Comment:
Might (?) work without the href, which is implicitly used whenever a URL is used as a string.

Reply:
That gave me
```
Error: sprite: string expected, object found
```
Not sure if there's a JS equivalent to Java's Object.toString()

Reply:
[Yes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/toString), and [URL overrides it](https://developer.mozilla.org/en-US/docs/Web/API/URL/toString). But `href` is fine too.

Reply:
Ah, never mind then.

(You could say `new URL(…) + ''` e.g. but that would just be less clear.)